src/printing.py

```diff
@@ -1,17 +1,17 @@
 """
 Python provides a number of ways to perform printing. Research
-how to print using the printf operator, the `format` string 
+how to print using the printf operator, the `format` string
 method, and by using f-strings.
 """
 
 x = 10
 y = 2.24552
 z = "I like turtles!"
-
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
-
+print('%d is 10, %.2f is 2.25, %s is "I like turtles!"' % (x, round(y, 2), z))
```

Comment:
This is similar to what mine looks like. I think %.2f and round() are redundant.